app/src/main/res/layout-sw600dp-port/ongoing_story_card.xml

```diff
@@ -54,26 +54,24 @@
 
         <TextView
           android:id="@+id/story_name_text_view"
+          style="@style/Body"
           android:layout_width="match_parent"
-          android:layout_height="wrap_content"
           android:layout_marginStart="8dp"
           android:layout_marginTop="8dp"
           android:layout_marginEnd="8dp"
           android:ellipsize="end"
-          android:fontFamily="sans-serif"
           android:lines="2"
           android:text="@{viewModel.ongoingStory.storyName}"
           android:textColor="@color/oppiaPrimaryText"
```

Comment:
`android:textColor="@color/oppiaPrimaryText"`
You can remove this line too because this is already mentioned in styles.xml

Check this:
`Body -> TextView.Common1`